src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -237,91 +241,86 @@
         }
 
         {% if excluded_ajax_paths is defined %}
-            if (window.fetch && window.fetch.polyfill === undefined) {
```

Comment:
please keep the indentation inside the Twig ``if``. It makes the template much more readable